### PR TITLE
📝(project) improve project description and quick start guide

### PR DESCRIPTION
**Pull request by jmaupetit**
## Purpose

Ralph is a complex objects: as a toolbox, it can be used in various ways. The project's documentation does not reflect possible usages.

## Proposal

Better describe core features and installation schemes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,31 @@
...
+$ pip install \
+    ralph-malph[backend-es,backend-ldp,backend-mongo,backend-swift,backend-ws,cli,lrs]
+
+# Install only the core package (library usage)
 $ pip install ralph-malph
 ```
 
-Once installed, the `ralph` command should be available in your `PATH`. Try to
+If you installed the full package (including the CLI, LRS and supported
+backends), the `ralph` command should be available in your `PATH`. Try to
```
Comment (SergioSim) [inline README.md:28]: It seems the `ralph` command is available even if only the core package is installed

**Review by SergioSim — COMMENTED** [review]
Looks great)

File: README.md (hunk 1 above)
Comment (jmaupetit) [reply README.md:28]: True, and it fails as it requires sentry + click. Maybe we should make sentry a CLI **and** a LRS dependency, and also better integrate sentry with the LRS server. I'll handle this in another PR.

File: README.md [hunk 2]
```diff
@@ -1,19 +1,31 @@
...
 Ralph is distributed as a standard python package; it can be installed _via_
 `pip` or any other python package manager (_e.g_ Poetry, Pipenv, etc.):
 
-```
+```sh
+# Install the full package
+$ pip install \
+    ralph-malph[backend-es,backend-ldp,backend-mongo,backend-swift,backend-ws,cli,lrs]
+
+# Install only the core package (library usage)
```
[the remark this answers is not in the document]
Comment (quitterie-lcs) [reply README.md:23]:
```suggestion
# Install only the core package (library usage)
```
```suggestion
# Install only the core package (library usage without backends, CLI and LRS)
```

[the remark this answers is not in the document]
Comment (jmaupetit) [reply README.md:23]: Well spotted thanks!

File: docs/index.md [hunk 3]
```diff
@@ -1,14 +1,41 @@
...
+- **HTTP API server**, to collect xAPI statements (learning events)
+  following the [ADL LRS
+  standard](https://github.com/adlnet/xAPI-Spec/blob/master/xAPI-Communication.md#partthree).
 
-Ralph has been designed to batch process your logs using base commands and UNIX
-standard streams (`stdin`, `stdout`) to connect them in a pipeline that fits
-your needs. A base example pipeline may be:
+## Supported backends
+
+As a library or a CLI, you will be able to read/write from/to the following
```
Comment (quitterie-lcs) [inline docs/index.md:15]:
```suggestion
As a library or a CLI, you will be able to read and write from/to the following
```

File: docs/index.md [hunk 4]
```diff
@@ -1,14 +1,41 @@
...
+traces in xAPI format:
+
+- Elasticsearch
+- MongoDB
+
+## CLI key concepts
+
+At first, Ralph has been designed as a CLI to batch process your logs using
+base commands and UNIX standard streams (`stdin`, `stdout`) to connect them in
+a pipeline that fits your needs. A base example pipeline may be:
```
Comment (quitterie-lcs) [inline docs/index.md:38]:
```suggestion
At first, Ralph has been designed as a CLI to batch process your logs using
base commands and UNIX standard streams (`stdin`, `stdout`) and connect them in
a pipeline. A base example pipeline may be:
```

**Review by quitterie-lcs — APPROVED** [review]

**Review by SergioSim — APPROVED** [review]
This is great)

**Pull request merged**